wiki/BanchoBot/id.md

```diff
@@ -96,7 +96,7 @@ Daftar perintah BanchoBot:
 !where <nama pemain>
 ```
 
-`!where` menampilkan lokasi pemain saat ini. Secara umum perintah ini akan menampilkan lokasi dari pemain. Tapi jika seorang pemain memiliki opsi `Tampilkan lokasi kota kamu kepada pengguna lain` yang telah dihidupkan, hal ini juga akan menampilkan kota di mana mereka saat ini aktif. Berikut contoh hasil dari perintah tersebut:
+`!where` menampilkan lokasi pemain saat ini. Secara umum, perintah ini akan menampilkan asal negara pemain. Tapi jika seorang pemain mengaktifkan opsi `Tampilkan lokasi kota kamu kepada pengguna lain`, maka asal kota mereka juga aakan ditampilkan. Berikut contoh hasil dari perintah tersebut:
```

Comment:
should be `Tampilkan lokasi kota Anda kepada pengguna lain`

![image](https://user-images.githubusercontent.com/36564236/105628451-9b9bec80-5e6f-11eb-9667-b4abd9541411.png)